tests/unittests/audio/test_nisqa.py

```diff
@@ -0,0 +1,208 @@
+# Copyright The Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+from typing import Any, Dict, Tuple
+
+import pytest
+import torch
+from torch import Tensor
+from torchmetrics.audio.nisqa import NonIntrusiveSpeechQualityAssessment
+from torchmetrics.functional.audio.nisqa import non_intrusive_speech_quality_assessment
+from torchmetrics.utilities.imports import _LIBROSA_AVAILABLE
+
+from unittests._helpers.testers import MetricTester
+
+if _LIBROSA_AVAILABLE:
+    import librosa
+else:
+    librosa = None
```

Comment:
```suggestion

from unittests._helpers.testers import MetricTester
```
the import does not seem to be used during the testing?

Reply:
True